### ISSUE TITLE
Not able to delete components

### ISSUE THREAD

**Issue by jamway** [Cloud-Pipelines/pipeline-editor#1]
### **Issue Description**

After dropping the component on the canvas I was not able to remove it from the canvas either by hitting delete key or trying to drag it off to the component selector
 

**Expected behavior**
After selecting the component to be deleted, the user can remove it from the canvas either by hitting the delete key or drag it off to the component selector.

----
Thanks for putting this together.
Please let me know if I am missing something obvious.

**Comment by Ark-kun** [comment]
Thank you for the feedback. I've realized the issue the day I've sent the announcement.
I understand the frustration. When I first tried react-flow I was surprised that the deletion did not work (https://github.com/wbkd/react-flow/issues/1268) and multi-selection did not work as well (https://github.com/wbkd/react-flow/issues/1269, https://github.com/wbkd/react-flow/issues/1270)  
Turns out the react-flow had Mac-only configuration that was using the Meta key for multi-selection (which translated to the Win key on Windows) and backspace for deletion.
I've changed those defaults to Delete for deletion, Ctrl for multi-selection and Shift for area selection, created a PR to add support for multiple keys and forgot about it (https://github.com/wbkd/react-flow/pull/1281). Turns out it's not merged yet.

I'll work on working around this issue ASAP.
